lambda_function.py

```diff
@@ -0,0 +1,49 @@
+import json
+import os
+import logging
+from google.cloud import bigquery
+
+
+def lambda_handler(event):
+    messageBody = event["Records"]
+
+    for eachMessage in messageBody:
+
+        # extracting the body of the message
+        message = json.loads(eachMessage["Sns"]["Message"])[0]
+
+        row = {}
+
+        # the key values are the column names in the BQ table.
+        row["timestamp"] = message["dateTime"]
+        row["purpose_type"] = message["purpose"]["type"]
+        row["purpose_sub_type"] = message["purpose"]["subType"]
+        row["platform"] = message["purpose"]["params"]["platform"]
+        row["platform_id"] = message["purpose"]["params"]["id"]
```

Comment:
This seems to be assuming none of these are optional. If so, you should quickly check if they exist. 

maybe something like this: https://stackoverflow.com/questions/1285911/how-do-i-check-that-multiple-keys-are-in-a-dict-in-a-single-pass

Reply:
Agreed. When we're sending these from the frontend, we're not checking there if something is undefined or not. So we should check here ideally and throw an error. And what should be done as an error case? @pritamps 
Should we just discard the sqs message right?